setup.py

```diff
@@ -16,7 +16,8 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    # the 'pre' suffix is needed for supporting '1.0b*' versions
```

Comment:
I develop on py3.6, pip 9.0.1. Tested to work on py3.5 with pip 7.0.0.

Looking at the included libraries it will not work with pip 6.0.